scaleway/resource_lb_certificate_beta_test.go

```diff
@@ -0,0 +1,239 @@
+package scaleway
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccScalewayLbCertificateBeta(t *testing.T) {
+	/**
+	* Note regarding the usage of xip.io
+	* See the discussion on https://github.com/terraform-providers/terraform-provider-scaleway/pull/396
+	* Long story short, scaleway api will not permit you to request a certificate in case common name is not pointed
+	* to the load balancer ip (which is unknown before creating it). In production this can be overcome by introducing
+	* an additional step which creates a dns record and depending on it, but for test purposes xip.io was ideal solution.
+	 */
```

Comment:
```suggestion
	/**
	* Note regarding the usage of xip.io
	* See the discussion on https://github.com/terraform-providers/terraform-provider-scaleway/pull/396
	* Long story short, scaleway API will not permit you to request a certificate in case common name is not pointed
	* to the load balancer IP (which is unknown before creating it). In production, this can be overcome by introducing
	* an additional step which creates a DNS record and depending on it, but for test purposes, xip.io is an ideal solution.
	*/
```